File: src/sdk-reference/php/3/memory-storage/hmset/snippets/hmset-1.php

<?php

use \Kuzzle\Kuzzle;


$kuzzle = new Kuzzle('localhost');

$entries = [
  ['field' => 'field1', 'value' => 'foo'],
  ['field' => 'field2', 'value' => 'bar'],
  ['field' => '...', 'value' => '...']
];

try {
  $kuzzle->memoryStorage()->hmset('key', entries);
}
catch (ErrorException $e) {

}

File: src/sdk/android/3/core-classes/memory-storage/hmset/snippets/hmset-1.java

JSONObject[] entries = new JSONObject[]{
  new JSONObject().put("field", "field1").put("value", "foo"),
  new JSONObject().put("field", "field2").put("value", "bar"),
  new JSONObject().put("field", "...").put("value", "...")
};

kuzzle.memoryStorage.hmset("key", entries, new ResponseListener<Void>() {
  @Override
  public void onSuccess(Void v) {
    // callback called once the action has completed
  }

  @Override
  public void onError(JSONObject error) {
  }
});
